java-numbers/src/test/java/com/baeldung/formatNumber/FormatNumberUnitTest.java

```diff
@@ -0,0 +1,37 @@
+package com.baeldung.formatNumber;
+
+import org.junit.Test;
+
+import static com.baeldung.formatNumber.FormatNumber.*;
+import static org.assertj.core.api.Assertions.*;
+
+public class FormatNumberUnitTest {
+        private static final double D = 4.2352989244d;
+        private static final double F = 8.6994540927d;
+
+        @Test public void givenDecimalNumber_whenFormatNumberToNDecimalPlaces_thenGetExpectedResult() {
```

Comment:
Let's move the method below the `@Test` annotation

Reply:
Sure